app/models/transaction.rb

```diff
@@ -0,0 +1,5 @@
+class Transaction < ApplicationRecord
+  enum result: [:failure, :success]
```

Comment:
if you called transaction.result! here, the status enum value would be 0? and 1 for success?

Reply:
Yep, we added a converter in the rake commands to swap them on pull from the csv to the integers. Should make for cleaner code later.